examples/sagemaker-tensorflow/README.md

```diff
@@ -0,0 +1,44 @@
+# Training and Serving Merlin on AWS SageMaker
+
+With AWS Sagemaker, you can package your own models that can then be trained
+and deployed in the SageMaker environment.
+The following notebook will show you how to use NVIDIA Merlin for training and
+inference in the SageMaker environment.
+
+- [Training and Serving Merlin on AWS SageMaker](sagemaker-merlin-tensorflow.ipynb)
+
+It assumes that readers are familiar wtth some basic concepts in NVIDIA Merlin,
```

Comment:
nit: s/It/This notebook/ && s/wtth/with/

The rationale is that there's a teensy bit of mental load to remember what the "it" pronoun refers to.  By restating the noun, the load is removed--however small.  It should feel simpler and easier to read.